From: Harald Sitter <sitter@kde.org>
Date: Mon, 05 Oct 2015 13:14:00 +0000
Subject: Install emotes and mimtetypes directories of the dark icon theme
X-Git-Url: http://quickgit.kde.org/?p=breeze.git&a=commitdiff&h=12d8b9a9907b28ba1cbaffc380f4eeae4de9774d
---
Install emotes and mimtetypes directories of the dark icon theme

CCMAIL: kainz.a@gmail.com
---


--- a/icons-dark/CMakeLists.txt
+++ b/icons-dark/CMakeLists.txt
@@ -11,7 +11,7 @@
 
 ########### install files ###############
 
-set( breeze_icon_dark_dirs  actions apps categories devices emblems places status)
+set( breeze_icon_dark_dirs  actions apps categories devices emblems emotes mimetypes places status)
 
 
 set(BREEZE_INSTALL_DIR ${ICON_INSTALL_DIR}/breeze-dark)

